BangazonWebApp/BangazonWebApp/BangazonApi.sql

```diff
@@ -223,6 +223,11 @@ CREATE TABLE PaymentTypes (
     AccountNumber INTEGER NOT NULL,
 	CustomersId INTEGER NOT NULL,
     CONSTRAINT FK_Customers FOREIGN KEY(CustomersId) REFERENCES Customers(Id),
+<<<<<<< HEAD
+	IsDeleted BIT NOT NULL
```

Comment:
This looks like an unresolved merge conflict. Did you mean to change the SQL file?